deploy.sh

```diff
@@ -13,7 +13,8 @@ if [[ "$TRAVIS_PULL_REQUEST" != "false" || "$TRAVIS_BRANCH" != "$SOURCE_BRANCH"
   echo "Skipping deploy; just doing a build."
   mkdir out
   doCompile
-  exit 0
+  node ./check-grammar.js ./out/index.html
+  exit $?
```

Comment:
Why only check on pull requests?

I'd just put this inside `doCompile`. No need for `exit $?` then since we have done `set -e` above.